docs/products/compute/compute-instances/guides/lish/index.md

```diff
@@ -175,31 +174,21 @@ ssh -t [manager-username]@lish-[location].linode.com [linode-name]
 
 You can also append Lish commands to the SSH command on your system prompt. For instance, to reboot your system, using your Cloud Manager username, location, and the host-id for your Compute Instance:
 
-    ssh -t [manager-username]@lish-[location].linode.com [linode-name] reboot
+```command
+ssh -t [manager-username]@lish-[location].linode.com [linode-name] reboot
+```
 
 Similarly, you can generate a view of the log using Lish:
 
-    ssh -t [manager-username]@lish-[location].linode.com [linode-name] logview
+```command
+ssh -t [manager-username]@lish-[location].linode.com [linode-name] logview
+```
 
 This command format works for all Lish functionality.
 
 ## Lish Gateways
 
-Each data center has its own gateway, which provides access to Lish, Weblish, and Glish. You can use any gateway to access your Compute Instances, but we recommend using one close to the data center it’s located within. The gateway boxes are available over IPv4 and IPv6.
-
-| Data center | Lish SSH | Weblish and Glish |
-| -- | -- | -- |
-| Atlanta (Georgia, USA) | `lish-atlanta.linode.com` | `atlanta.webconsole.linode.com` |
-| Dallas (Texas, USA) | `lish-dallas.linode.com` | `dallas.webconsole.linode.com` |
-| Frankfurt (Germany) | `lish-frankfurt.linode.com` | `frankfurt.webconsole.linode.com` |
-| Fremont (California, USA) | `lish-fremont.linode.com` | `fremont.webconsole.linode.com` |
-| London (United Kingdom) | `lish-london.linode.com` | `london.webconsole.linode.com` |
-| Mumbai (India) | `lish-mum1.linode.com` | `mum1.webconsole.linode.com` |
-| Newark (New Jersey, USA) |  `lish-newark.linode.com` | `newark.webconsole.linode.com` |
-| Singapore | `lish-singapore.linode.com` | `singapore.webconsole.linode.com` |
-| Sydney (Australia) | `lish-syd1.linode.com` | `syd1.webconsole.linode.com` |
-| Tokyo (Japan) | `lish-tokyo2.linode.com` <br>or `lish-shg1.linode.com` | `shg1.webconsole.linode.com` |
-| Toronto (Canada) | `lish-tor1.linode.com` | `tor1.webconsole.linode.com` |
+Each data center has its own gateways, which provides access to Lish, Weblish, and Glish. When connecting through Lish, To connect to one of these services, use the corresponding gateway in any data center, but we recommend using one close to the data center it’s located within. The gateway boxes are available over IPv4 and IPv6.
```

Comment:
lowercase `to`, but one of those sentence intros could be removed. Either `When connecting through Lish,` or  `To connect to one of these services,`

Also, I think `but` can be removed and what follows turned into its own sentence.

Reply:
I reworked this paragraph to fix these issues.